crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -392,6 +405,12 @@ fn write_cheatcode_panic(buffer: &mut MemBuffer, panic_data: &[Felt252]) {
         .expect("Failed to insert error in memory");
 }
 
+fn contract_address_from_felt252(felt: &Felt252) -> Result<ContractAddress, EnhancedHintError> {
+    Ok(ContractAddress(PatriciaKey::try_from(StarkFelt::new(
+        felt.to_be_bytes(),
```

Comment:
I think we can use blockifier's `felt_to_stark_felt`

Reply:
You're totally right. Thanks!